ethmonitor/ethmonitor_test.go

```diff
@@ -267,3 +267,24 @@ func TestMonitorWithReorgme(t *testing.T) {
 
 	monitor.Stop()
 }
+
+func XXTestMonitorAverageBlockTime(t *testing.T) {
```

Comment:
just wondering why this test case is disabled? no worries if you want to keep it disabled and using just for debugging, but checking in

Reply:
I added `time.Sleep(10 * time.seconds)` to get some blocks for calculation. I don't want our ga to wait for 10 seconds for this test.